### PR TITLE
Add a script and Github action to check a PRs age

### PR DESCRIPTION
**Pull request by pezholio**
By convention, we shouldn't adopt an RFC for two weeks (even if it's recieved enough feedback), to give all team members a chance to wade in. A couple of PRs have been merged before this time limit though, so this adds a simple script to check the age of a PR and set a status. If a PR is under two weeks old, the status is set to `pending`, if it's over two weeks old, the status is set to `passed`. This should be enough to prevent an accidental merge before the time has elapsed.

The wording is a bit clumsy on the statuses, so I'd welcome some feedback on a better form of words.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,14 +3,16 @@
   "private": true,
   "scripts": {
     "lint": "remark --frail .",
-    "lint:fix": "remark --output --frail ."
+    "lint:fix": "remark --output --frail .",
+    "check_prs": "node ./utils/check_prs.js"
```
Comment (erbridge) [inline package.json:7]:
```suggestion
    "check-prs": "node ./utils/check-prs.js"
```

It's confentional to use dashes and camelCase for JavaScript.

**Comment by erbridge** [comment]
Made some tweaks to this and updated the wording. A feature that would be nice to have would be for it to look for some keyphrase in the PR title for tooling updates (like this one).

**Comment by erbridge** [comment]
The promise handling could do with some love (any errors will be swallowed by the `.then`), but I don't think that blocks it.

**Comment by pezholio** [comment]
> A feature that would be nice to have would be for it to look for some keyphrase in the PR title for tooling updates (like this one)

Sorted! The script now skips any PRs with a `tooling` label applied to them. I've also documented what we do in the README 👍

**Comment by pezholio** [comment]
If there are no objections, I'm a gonna merge this. The action is running anyhoo

**Pull request merged**